doc/DISCLAIMER.md

```diff
@@ -0,0 +1,7 @@
+* Known limitations:
+    * Are LDAP and HTTP auth supported? LDAP=Yes | HTTP auth=No
+    * Can the app be used by multiple users? Yes
+
+
+* Additional informations:
+    * As the pyhton version shipped in Debian stable is not always supported, a recent version could be built during the installation process. It may take a while to achive that (15 to 60 minutes)
```

Comment:
```suggestion
Known limitations:
* LDAP login is supported, but not single sign-on
* As the pyhton version shipped in Debian stable is not always supported, a recent version could be built during the installation process. It may take a while to achieve that (15 to 60 minutes)
```